src/main/java/me/karakelley/tictactoe/ImpossibleBehavior.java

```diff
@@ -0,0 +1,53 @@
+package me.karakelley.tictactoe;
+
+import java.util.*;
+
+public class ImpossibleBehavior implements AIBehavior {
+  private String[] players;
+  private Game game;
+
+  public ImpossibleBehavior(String opponentMarker, String computerMarker, Game game) {
```

Comment:
Does the impossibleAi need to be passed in the opponentMarker? Or the current marker? Can't it infer whose turn it is from the state of the board or ask the Game?